src/web_app/js/sdputils.js

```diff
@@ -34,6 +35,20 @@ function iceCandidateType(candidateStr) {
   return candidateStr.split(' ')[7];
 }
 
+// Turns the local type preference into a human-readable string.
+// Note that this mapping is browser-specific.
+function formatTypePreference(pref) {
```

Comment:
good question... I don't think it is generally useful enough, but it should have a switch for Firefox support here